subprojects/capstone-v4.wrap

```diff
@@ -1,6 +1,5 @@
-[wrap-file]
-source_url = https://github.com/capstone-engine/capstone/archive/4.0.2.tar.gz
-source_filename = 4.0.2.tar.gz
-source_hash = 7c81d798022f81e7507f1a60d6817f63aa76e489aa4e7055255f21a22f5e526a
+[wrap-git]
+url = https://github.com/capstone-engine/capstone.git
```

Comment:
What is the plan for those? Asking capstone devs to make 4.0.3 and 5.0.2 releases?

Reply:
Yes, Though I think @kabeor wants to release it with the pre-release together?